lib/prometheus_exporter/instrumentation/sidekiq.rb

```diff
@@ -9,21 +9,42 @@ def initialize(client: nil)
 
     def call(worker, msg, queue)
       success = false
+      shutdown = false
       start = ::Process.clock_gettime(::Process::CLOCK_MONOTONIC)
       result = yield
       success = true
       result
+    rescue ::Sidekiq::Shutdown => e
+      shutdown = true
+      raise e
     ensure
       duration = ::Process.clock_gettime(::Process::CLOCK_MONOTONIC) - start
       class_name = worker.class.to_s == 'ActiveJob::QueueAdapters::SidekiqAdapter::JobWrapper' ?
                      msg['wrapped'] : worker.class.to_s
-
       @client.send_json(
         type: "sidekiq",
         name: class_name,
         success: success,
+        shutdown: shutdown,
+        retries_exhausted: retries_exhausted?(msg),
         duration: duration
       )
     end
+
+    private
+
+    def retries_exhausted?(msg)
```

Comment:
FYI: If you add a feature that is not specified in the card it would be great to add some motivation. I mean it is pretty obvious here but still 😸